basculehttp/provide_test.go

```diff
@@ -111,6 +130,8 @@ capabilities:
      - "aaaa\\b"
      - "bbbn/.*\\b"
 `,
+// TODO ask wes how to setup goschtalt too allow incomplete configs
```

Comment:
@schmidtw is there a way to setup goschtalt to allow incomplete configs? this test case is failing because goschtalt expects to find `capabilities`

Reply:
You can mark the Unmarshal() operation with the Optional() option.  https://pkg.go.dev/github.com/goschtalt/goschtalt#Optional ... but should bascule include a configuration library like goschtalt, or should it instead define configuration structures that can be unmarshaled by any configuration library?

Reply:
I think `bascule` shouldn't itself unmarshal anything, which in turn would mean it wouldn't have a dependency on `goschtalt`.  But, I realize that might be a bigger change atm.  We could move forward in increments.

Reply:
cool 🙂 
I'm assuming our future basecule rewrite is also not going to include a configuration library.
since I'm already removing arrange which include `arrange.UnmarshalKey`, I think we should go ahead and remove it now... making the transition to the new bascule will be little easier

Reply:
done!